command/node_status.go

```diff
@@ -289,8 +289,14 @@ func (c *NodeStatusCommand) Run(args []string) int {
 				out[i+1] += fmt.Sprintf("|%s", node.Attributes["os.name"])
 			}
 			if c.verbose {
-				out[i+1] += fmt.Sprintf("|%s|%s",
-					node.Address, node.Version)
+				var nodeMaxAlloc string
+				if node.NodeMaxAllocs == 0 {
+					nodeMaxAlloc = "N/A"
+				} else {
+					nodeMaxAlloc = strconv.FormatUint(node.NodeMaxAllocs, 10)
+				}
+				out[i+1] += fmt.Sprintf("|%s|%s|%s",
+					node.Address, node.Version, nodeMaxAlloc)
```

Comment:
There's a very small number of fields on this `nomad status` list. It doesn't seem like the max allocs allowed on a node rises to that level of importance? I'd leave it out of this output.